scripts/pre_leap.sh

```diff
@@ -19,23 +19,16 @@
 set -e -u
 set -o pipefail
 
+# Pre-flight check
```

Comment:
Make sure to pushd /opt/rpc-upgrades directory otherwise I think it'll have a problem finding the inventoryas the gate job is showing right now.  Other than that, lgtm.

Reply:
cool